app/build.gradle

```diff
@@ -107,7 +107,7 @@ dependencies {
       'androidx.multidex:multidex:2.0.1',
       'androidx.recyclerview:recyclerview:1.0.0',
       'androidx.work:work-runtime-ktx:2.4.0',
-      'com.chaos.view:pinview:1.4.3',
+      'com.chaos.view:pinview:1.4.4',
```

Comment:
Just to be sure, are we updating the version everywhere it is required including bazel files too?

Reply:
Honestly I have no idea what Bazel does, but I did find some places in the project where the version was 1.4.3 instead of 1.4.4, should I change that?

Reply:
yes